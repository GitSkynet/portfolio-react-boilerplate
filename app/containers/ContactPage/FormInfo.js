import styled from 'styled-components';

const FormInfo = styled.div`
  padding-bottom: 1rem;
`;

export default FormInfo;
