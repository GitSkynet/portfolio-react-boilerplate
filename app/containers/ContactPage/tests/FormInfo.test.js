import React from 'react';
import { shallow } from 'enzyme';

import FormInfo from '../FormInfo';

describe('FormInfo', () => {
  it('should render and match the snapshot', () => {
    const wrapper = shallow(<FormInfo />);
    expect(wrapper).toMatchSnapshot();
  });
});
