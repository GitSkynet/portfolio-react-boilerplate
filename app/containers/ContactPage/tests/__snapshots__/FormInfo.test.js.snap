// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInfo should render and match the snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="FormInfo__FormInfo-gAHIHB eQCYPD"
/>,
  "nodes": Array [
    <div
      className="FormInfo__FormInfo-gAHIHB eQCYPD"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <FormInfo__FormInfo />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "eQCYPD",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="FormInfo__FormInfo-gAHIHB eQCYPD"
/>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="FormInfo__FormInfo-gAHIHB eQCYPD"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <FormInfo__FormInfo />,
}
`;
